bindings/nodejs/tests/utils.mjs

```diff
@@ -17,33 +17,40 @@
  * under the License.
  */
 
-const path = require('path')
+import crypto from 'node:crypto'
 
 export function generateBytes() {
-    const size = Math.floor(Math.random() * 1024) + 1
-    const content = []
+  const size = Math.floor(Math.random() * 1024) + 1
+  const content = []
 
-    for (let i = 0; i < size; i++) {
-        content.push(Math.floor(Math.random() * 256))
-    }
+  for (let i = 0; i < size; i++) {
+    content.push(Math.floor(Math.random() * 256))
+  }
 
-    return Buffer.from(content)
+  return Buffer.from(content)
 }
 
 export function loadTestSchemeFromEnv() {
-    require('dotenv').config({ path: path.resolve(__dirname, '../../../.env'), debug: true })
-    return process.env.OPENDAL_TEST
+  return process.env.OPENDAL_TEST
+}
+
+export function checkRandomRootEnabled() {
+  return process.env.OPENDAL_DISABLE_RANDOM_ROOT !== 'false'
+}
+
+export function generateRandomRoot() {
+  return `/opendal_${crypto.randomUUID()}`
```

Comment:
We should respect the original root, for example: `{root}/opendal_${crypto.randomUUID()}`.

Reply:
Updated.